bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -51,80 +62,191 @@
+            Bundle bundle, TranslationProvider i18nProvider, LocaleProvider localeProvider) {
         super(bridge);
         this.httpClientFactory = httpClientFactory;
         this.webSocketFactory = webSocketFactory;
+        this.bundle = bundle;
+        this.i18nProvider = i18nProvider;
+        this.localeProvider = localeProvider;
+    }
+
+    /**
+     * Returns the time when the last api call was made. If the thing was newly created, restores the time from a
+     * property to ensure consistent behaviour over restarts.
+     *
+     * @return the time when the last api call was made.
+     */
+    private Instant lastApiCallTime() {
+        if (!lastApiCallTimeLoaded) {
+            Map<String, String> properties = getThing().getProperties();
+            String property = properties.getOrDefault(GardenaBindingConstants.LAST_API_CALL_TIME, "");
+            lastApiCallTime = "".equals(property) ? Instant.now().minus(Duration.ofHours(1)) : Instant.parse(property);
+            lastApiCallTimeLoaded = true;
+        }
+        return lastApiCallTime;
+    }
+
+    /**
+     * Updates the time when the last api call was made to now(). Saves the value to a property to ensure consistent
+     * behaviour over restarts.
+     */
+    private void lastApiCallTimeUpdate() {
+        lastApiCallTime = Instant.now();
+        getThing().setProperty(GardenaBindingConstants.LAST_API_CALL_TIME,
+                lastApiCallTime.truncatedTo(ChronoUnit.SECONDS).toString());
     }
 
     @Override
     public void initialize() {
         logger.debug("Initializing Gardena account '{}'", getThing().getUID().getId());
-        initializeGardena();
+        Instant now = Instant.now();
+        Instant notBeforeTime = lastApiCallTime().plus(REINITIALIZE_DELAY_MINUTES_BACK_OFF)
+                .plus(REINITIALIZE_DELAY_MINUTES_BACK_OFF);
+        if (now.isBefore(notBeforeTime)) {
+            // delay the initialisation
+            Duration delay = Duration.between(now, notBeforeTime);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, uiText(delay.getSeconds()));
+            scheduleReinitialize(delay);
+        } else {
+            // do immediate initialisation
+            scheduler.submit(() -> initializeGardena());
+        }
     }
 
     public void setDiscoveryService(GardenaDeviceDiscoveryService discoveryService) {
         this.discoveryService = discoveryService;
     }
 
+    /**
+     * Format a localised description text to display on the thing main UI page.
+     *
+     * @param delaySeconds the delay that will be added to the current time.
+     * @return the (localised) description text.
+     */
+    private String uiText(long delaySeconds) {
+        return i18nProvider.getText(bundle, RECONNECT_MSG_KEY, RECONNECT_MSG_KEY, localeProvider.getLocale())
+                + LocalDateTime.now().plusSeconds(delaySeconds).format(DATE_TIME_FORMAT);
```

Comment:
`LocalDateTime.now()` will use default time-zone, not time-zone configured in openHAB. Since it's just a text string for the offline status, you can leave it as is, but if you would like to display the local time for the configured time-zone, you should use [TimeZoneProvider](https://www.openhab.org/javadoc/latest/org/openhab/core/i18n/timezoneprovider). You can then use `LocalDateTime.now(timeZoneProvider.getTimeZone())`.

Reply:
Ok. Done.